test/test_parser.py

```diff
@@ -151,15 +151,16 @@ def testObjColors(self):
 
 class TestMtlParser(unittest.TestCase):
     def setUp(self):
-        # Append current path to locate files
-        meshes = pywavefront.Wavefront(prepend_dir('simple.obj'))
-        self.material1 = meshes.mesh_list[0].materials[0]
-        self.material2 = meshes.mesh_list[1].materials[0]
+        parser = MaterialParser(prepend_dir('simple_parsetest.mtl'))
+        self.materials = parser.materials
+        self.material1 = self.materials['Material.simple']
+        self.material2 = self.materials['Material2.simple']
 
     def testMtlName(self):
         """Parsing an obj file with known material names should set those names."""
-        self.assertEqual(self.material1.name, 'Material.simple')
-        self.assertEqual(self.material2.name, 'Material2.simple')
+        print(self.materials)
```

Comment:
Should this be a `log` instead of a `print`?

Reply:
Removing this